src/App.jsx

```diff
@@ -14,70 +16,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [currentGood, setCurrentGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {currentGood !== ' ' ? (
+        <h1 className="title is-flex is-align-items-center">
+          {`${currentGood} is selected`}
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              setCurrentGood(' ');
+            }}
```

Comment:
```suggestion
            onClick={reset}
```
Consider moving this function from JSX and reusing it further